_roadmap/03_submissions.md

```diff
@@ -17,8 +17,8 @@ Previously, we allowed participants to submit multiple sets of predictions (for
 Partly, this was to ensure optimal opportunities for learning from participation; participants could run several methods to compare and contrast their results.
 
 However, for SAMPL7 onwards, we are modestly changing our policy on multiple submissions to distinguish two categories of submissions:
-1) **Challenge submissions**: Formal, judged entries in SAMPL (only one per research group or research organization)
-2) **Verified submissions**: Blind predictions (unlimited per research group or research organization) which *are not formally judged*.
+1. **Challenge submissions**: Formal, judged entries in SAMPL (only one per research group or research organization)
+2. **Verified submissions**: Blind predictions (unlimited per research group or research organization) which *are not formally judged*.
```

Comment:
```suggestion
2. **Verified submissions**: Blind predictions (unlimited per research group or research organization) which *are not formally judged*. Evaluation statistics will still be provided for comparison of verified submissions.
```

Reply:
I like the separation of submissions into two categories. I think it will allow easier evaluations in the future. But I don't think "challenge submission" is a clear enough term. All blind submissions are submissions to the challenge after all. Would you consider renaming this in a more clear way? "Competitive submissions" or "Ranked submissions" comes to my mind.  I like "competitive submission" more.

Reply:
OK, I'll go with "Ranked submissions". I want to downplay the competitive aspect a bit.